packages/flutter_markdown/README.md

```diff
@@ -0,0 +1,12 @@
+# Flutter Markdown
+
+A markdown renderer for Flutter. It supports the [original format](https://daringfireball.net/projects/markdown/), but no inline html.
+
+## Getting Started
+
+Using the Markdown widget is simple, just pass in the source markdown as a string:
+
+    new Block(
```

Comment:
Perhaps we should have two widgets: Markdown and MarkdownBody.  That way Markdown handles the common case of wanting a scrollable container for the markdown.  Markdown can be built out fo MardownBody.  That's how we approach this issue elsewhere in the framework.